README.md

```diff
@@ -3,6 +3,7 @@
 
 [![npm](https://img.shields.io/npm/v/@finos/perspective.svg?style=flat)](https://www.npmjs.com/package/@finos/perspective)
 [![PyPI](https://img.shields.io/pypi/v/perspective-python.svg?style=flat)](https://pypi.python.org/pypi/perspective-python)
+![Conda](https://img.shields.io/conda/v/conda-forge/perspective)
```

Comment:
👎

No more badges please.

Reply:
Booo! More badges! I'll update the docs instead to mention conda, probably worth it as e.g. Mac arm and python 3.10 builds are already available there while not yet available as wheels on pypi

Reply:
fixed